diff --git a/lib/types/sharedOptions.d.ts b/lib/types/sharedOptions.d.ts
index a06f9d2fcdd26a339579fca35723d415a3ae3868..6837cad7312991e07ed511677d6376df1363af75 100644
--- a/lib/types/sharedOptions.d.ts
+++ b/lib/types/sharedOptions.d.ts
@@ -20,4 +20,4 @@ export interface LifeCycleEventsMap<ResponseType> {
     'response:mocked': (response: ResponseType, requestId: string) => void;
     'response:bypass': (response: ResponseType, requestId: string) => void;
 }
-export declare type LifeCycleEventEmitter<ResponseType> = Pick<StrictEventEmitter<ResponseType>, 'on' | 'removeListener' | 'removeAllListeners'>;
+export declare type LifeCycleEventEmitter<ResponseType extends Record<string | symbol, any>> = Pick<StrictEventEmitter<ResponseType>, 'on' | 'removeListener' | 'removeAllListeners'>;